sdk/cosmosdb/cosmos/package.json

```diff
@@ -148,7 +149,7 @@
       "QueryThroughput.ts",
       "SasTokenAuth.ts",
       "ServerSideScripts.ts",
-      "Shared/handleError.ts"
+      "handleError.ts"
```

Comment:
why it's moved out of shared?

Reply:
The actual handleError.ts file is still under Shared/ directory. The matcher of dev-tools was not handling full path. hence I had to update it to only filename.